shared/prometheus/service_test.go

```diff
@@ -84,3 +84,11 @@ func TestHealthz(t *testing.T) {
 	}
 
 }
+
+func TestStatus(t *testing.T) {
+	fStatus:= s.failStatus
```

Comment:
@cyberbono3 There is no `s` here , so you are getting a compile error

Reply:
How about you do this:
```go
failure := "something really bad happened"
s := &Status{failStatus: failure}

...
```